apps/backend/models/user.ts

```diff
@@ -27,21 +35,43 @@ export const User = list({
       isIndexed: 'unique',
     }),
     password: password({
+      //keystone password field automatically hashes password
       validation: {
         length: { min: 10, max: 100 },
         isRequired: true,
         rejectCommon: true,
       },
       bcrypt: require('bcryptjs'),
     }),
+    avatarUrl: text({
+      validation: {
+        match: { regex: /^https?:\/\/.+/i, explanation: 'Avatar must be a valid URL' }, // ensure that the string contains http:// or https:// and at least one letter after
+      },
+    }),
+    age: integer({
+      validation: { isRequired: true },
+    }),
+    gender: select({
+      options: [
+        { label: 'Female', value: 'female' },
+        { label: 'Male', value: 'male' },
+        { label: 'Non-Binary', value: 'non-binary' },
+      ],
+      validation: { isRequired: true },
```

Comment:
As of now adding the required columns of `age` and `gender` without a default value breaks the signup flow for new users since we don't have the frontend input fields for them

It also breaks the migrations for existing data since it will fail if there are existing users in the database

My recommendation is instead of adding a default value, to make these fields optional initially, and we will come back and refactor them to be required later

```suggestion
      validation: { isRequired: false },
```